components/scream/src/physics/p3/p3_functions_f90.hpp

```diff
@@ -162,6 +162,79 @@ void access_lookup_table_coll_f(Int dumjj, Int dumii, Int dumj, Int dumi, Int in
 
 }
 
+struct CloudWaterConservationData
+{
+  // inputs
+  Real qc;
```

Comment:
I would stick to the pattern I've established in the other *Data structs of doing multi-decls per line.